modules/@apostrophecms/widget-type/index.js

```diff
@@ -161,6 +161,14 @@ module.exports = {
           ...self.getWidgetsBundles(`${widget.type}-widget`)
         };
 
+        if (widget.aposPlaceholder === true) {
```

Comment:
I think it's still best to do `output = { ... output };` here. That is enough to prevent an issue if someone gets clever and does a save operation inside an async component in a widget template.